### PR TITLE
State of mind and employee assistance

### PR DESCRIPTION
**Pull request by lukemorton**
Hi all,

We want to make our state of mind a first class concern at Made Tech. We want to be happy but we also need to support each other when we're not. To this end we've setup an employee assistance programme to provide access to self-service counselling and we've also updated our literature around sickness to include stress, bereavement, arguments, etc.

In order to provide better support to us all, we're in the process of setting up the employee assistance programme. This programme will be able to provide phone counselling to anyone for free along with free legal/hr/financial advice, please read the document for more information. This programme goes live from 1st June.

Feedback more than welcome.

Luke

### CODE REVIEW

**Comment by Bennell** [comment]
No feedback other than this is a great idea. Like we talked about last week, the idea that having a cold is seen as a more valid illness than a host of mental illnesses (whether short term and temporary or longer term) is crazy. It's awesome to see MadeTech taking this stuff seriously and making efforts to improve quality of life for its employees. The more companies that do things like this, the faster the stigma of mental illness will hopefully start to fade away. Very proud to be part of it.

**Comment by lukemorton** [comment]
This PR should go someway to resolving #71.

**Comment by lukemorton** [comment]
Thanks Luke, so glad you feel proud to be a part of it. I do too :)

File: guides/welfare/state_of_mind.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Supporting each other
+
+We are a team and it is our duty to care for each other and ourselves, the latter being just as important as the former.
+
+- Take time off when you're not feeling great in the mind
+- Ask a colleague, friend, family member for support or lend an ear if you can, sharing helps
+- If a colleague looks like they're struggling or need help, be extra kind and supportive, ask if they need help
+- If you want confidential advice or phone counselling you can use our [employee assistance programme](employee_assistance.md)
+- If you feel like you're really in trouble, please ask a director or colleague for help, we can provide assistance in whatever shape it need take, from extending time off to providing a course of counselling
```
Comment (sowasred2012) [inline guides/welfare/state_of_mind.md]: I feel like this point, the second and the third are the same (ask for/ offer help).

Comment (lukemorton) [inline guides/welfare/state_of_mind.md]: Agree, but feel the need to elaborate around the fact we'll go the extra mile to help.

File: guides/welfare/employee_assistance.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- Legal advice (for personal and work)
+- Emotional help and counselling
+- External HR advice (completely unbiased help and support)
+- Health (pregnancy, sickness, personal injury, stress)
+- Family (rights, divorce, child support, teenagers)
+- Older people (bereavement, elderly care, support)
+- Financial advice (pay, pensions, tax, debt help, benefits)
+- Relationships (divorce & separation, counselling, bullying)
+- Addiction (substance and gambling, support for family, rehabillitation
```
Comment (zuren) [inline guides/welfare/employee_assistance.md]: rehabilitation*

File: guides/welfare/state_of_mind.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+We want every one to be happy, including our team. To that end we try and provide [flexible ways of working](https://github.com/madetech/handbook/tree/master/benefits) so that we can have a healthy work/life balance.
+
+Sometimes though, personal circumstances, stress at work, and other factors can affect our state of mind. Rather than consider this shameful, or brushing it under the carpet, we want to recognise that we aren't always going to be happy and nor should we be expect to be.
+
+## Supporting each other
+
+We are a team and it is our duty to care for each other and ourselves, the latter being just as important as the former.
+
+- Take time off when you're not feeling great in the mind
```
Comment (zuren) [inline guides/welfare/state_of_mind.md]: Could we put something more specific than "not great in the mind"?

Feel like this is pretty vague.

Comment (lukemorton) [inline guides/welfare/state_of_mind.md]: Open to suggestions :)

File: guides/welfare/parental_leave.md [hunk 4]
```diff
@@ -4,4 +4,4 @@ Our current parental leave package is two weeks at full pay followed by statutor
 
 In addition to this, we offer a number of benefits, including [Remote Working](benefits/remote_working.md) and [Untracked Holiday Allowance](benefits/untracked_holiday.md). We believe these benefits provide new parents with flexibility and allow them to achieve a good balance between work and family life. 
 
-_Note: There is a strong desire to improve our parental leave offering. We're currently constrained by our company size, but as we grow and get a larger team and turnover, we're expecting to be able to make a more attractive package available._
\ No newline at end of file
+_Note: There is a strong desire to improve our parental leave offering. We're currently constrained by our company size, but as we grow and get a larger team and turnover, we're expecting to be able to make a more attractive package available._
```
Comment (camillavk) [inline guides/welfare/parental_leave.md]: Just curious, by parental leave does this mean both paternity or maternity leave? I.e. are both of them the same (2 weeks?) and what does "followed by statutory allowance" actually mean? (Sorry, not related to your actual PR)

File: guides/welfare/state_of_mind.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+- Ask a colleague, friend, family member for support or lend an ear if you can, sharing helps
+- If a colleague looks like they're struggling or need help, be extra kind and supportive, ask if they need help
+- If you want confidential advice or phone counselling you can use our [employee assistance programme](employee_assistance.md)
```
Comment (adrianclay) [inline guides/welfare/state_of_mind.md]: I'd emphasise the confidentialness point on ` employee_assistance.md ` too.  Can't see it mentioned on the page.

File: guides/welfare/parental_leave.md (hunk 4 above)
Comment (lukemorton) [inline guides/welfare/parental_leave.md]: I'd love for you to open this as an issue if you'd like so we can continue convo there.

File: guides/welfare/state_of_mind.md (hunk 5 above)
Comment (lukemorton) [inline guides/welfare/state_of_mind.md]: Cheers will do

File: guides/welfare/state_of_mind.md (hunk 3 above)
Comment (Bennell) [inline guides/welfare/state_of_mind.md]: 'when you're not feeling great mentally'

Comment (sowasred2012) [inline guides/welfare/state_of_mind.md]: Something about taking time off to protect your mental wellbeing/health?

**Pull request merged**